contracts/plugins/assets/curve/CurveStableMetapoolCollateral.sol

```diff
@@ -124,8 +120,8 @@ contract CurveStableMetapoolCollateral is CurveStableCollateral {
     // Check for defaults outside the pool
     function _anyDepeggedOutsidePool() internal view virtual override returns (bool) {
         try this.tryPairedPrice() returns (uint192 low, uint192 high) {
-            // {UoA/tok} = {UoA/tok} + {UoA/tok}
-            uint192 mid = (low + high) / 2;
+            // D18{UoA/tok} = D18{UoA/tok} + D18{UoA/tok}
+            uint256 mid = (low + uint256(high)) / 2;
```

Comment:
why do we need this?

Reply:
See here: https://github.com/code-423n4/2023-07-reserve-findings/issues/21

Reply:
Hmm I guess now that RTokenAsset always returns (0, FIX_MAX) instead of (>0, FIX_MAX), then this isn't necessary. Still, seems better to have protection here and soft default instead of having refresh() revert.